UPGRADING.md

```diff
@@ -0,0 +1,19 @@
+## Upgrading Mongoid Locker
+
+### Upgrading to 1.0.0
+Locker is now doesn't define `:locked_at` and `:locked_until` fields by default during include.
```

Comment:
`Mongoid::Locker` no longer defines `locked_at` and `locked_until` fields when included. You must define these fields manually.